### PR TITLE
Spacing

### LINKED ISSUE #51: Action/Observation Spaces
Defines what what information space an action/observation can exist in. Inspired by the OpenAI Gym design >> https://gym.openai.com/docs/#spaces

### PR DESCRIPTION
**Pull request by SeverTopan**
Resolves #51. Just provides a lexical handle to carry around the idea of a spanning set of actions and observations which an agent may interface with.

### CODE REVIEW

File: Lexicon.md [hunk 1]
```diff
@@ -9,12 +9,14 @@
...
 * __Object__: A set of correlated __Observations__.
 * __System__: An __Object__ composed of correlated member __Objects__.
 * __Model__: A __System__ that represents another __System__. It can be assigned a truth value (measurement) which expresses the accuracy of the representation. Examples of __Models__ include maps, blueprints, simulations, mathematical models, beliefs and claims.
 * __World State__: Given that the __Information__ in an __Environment__ may change with respect to some parameter (for example, time), __World State__ defines a static snapshot of the __Information__ within an __Environment__. In other words, a __World State__ is an __Environment__ where all __Information__ is held constant.
 * __World Model__: An __Agent__'s internal __Model__ of its __World__.
 * __Action__: __Information__ that crosses the __Agent Boundary__, exiting the __Agent__.
+* __Action Space__: The set of all __Actions__ that a particular __Agent__ may perform. All __Actions__ percieved by an __Agent__ are sampled from its __Action Space__.
```
Comment (macterra) [inline Lexicon.md]: Should be "all Actions performed..."

File: Lexicon.md [hunk 2]
```diff
@@ -9,12 +9,14 @@
 * __Agent__: A __System__ that interacts with an __Environment__. It can consume zero or more __Observations__, and perform zero or more __Actions__. The decision of which __Actions__ to perform is made based on the __Agent's__ beliefs.
 * __Agent Boundary__: The surface that separates an __Agent__ from its __Environment__.
 * __Observation__: __Information__ that crosses the __Agent Boundary__, entering the __Agent__.
+* __Observation Space__: The set of all __Observations__ that a particular __Agent__ may observe. All __Observations__ percieved by an __Agent__ are sampled from its __Observation Space__.
```
Comment (macterra) [inline Lexicon.md]: Should be "All Oberservations perceived..."

**Review by macterra — CHANGES_REQUESTED** [review]
Typos

**Comment by SeverTopan** [comment]
Typos addressed

**Review by macterra — APPROVED** [review]

**Review by cecilyth — APPROVED** [review]

**Pull request merged**